dotCMS/src/main/java/com/dotmarketing/business/VersionableFactory.java

```diff
@@ -230,7 +243,7 @@ protected abstract ContentletVersionInfo createContentletVersionInfo(final Ident
 	 * @return
 	 * @throws DotDataException
 	 */
-	public abstract Optional<ContentletVersionInfo> findAnyContentletVersionInfo(final String identifier, final boolean deleted)
```

Comment:
I am not pretty sure if you should broke the code by renaming the method, probably would be better to introduce a new method and call it with a default value from the old findAnyContentletVersionInfo

Reply:
I am not renaming the method 


https://github.com/dotCMS/core/pull/24182/files#diff-59e693ff3b83a51f4ef792a6989091f119252f235210d01adc4f8dd9d1284d19R234

I am just creating a new one